tekton/release.sh

```diff
@@ -3,7 +3,7 @@ RELEASE_VERSION="${1}"
 COMMITS="${2}"
 
 UPSTREAM_REMOTE=upstream
-MASTER_BRANCH="master"
+MAIN_BRANCH="main"
```

Comment:
instead can we rename this to `DEFAULT_BRANCH` so that whenever default branch name changes(although it's not going to happen now) we need to change only the value of this variable and not change the variables in rest of the script. WDYT?

Reply:
Yes, it makes sense to me.